main.tf

```diff
@@ -2,7 +2,7 @@
 # Resource
 # ------------------------------------------------------------------------------
 resource "aws_acm_certificate" "main" {
-  domain_name       = "${var.domain}"
+  domain_name       = "${var.site}.${var.domain}"
```

Comment:
This means we can't have a TDL right? Not sure if its needed though.

Reply:
You can by setting site to "*" (which I think you should not do in addition to selecting create_wildcard). I will update the README to explain better

Reply:
I believe that setting it to wildcard will not allow my-domain.com, only service.my-domain.com. I guess we could solve it with some conditions but we can wait until there is a need with that.